src/routes/Login.jsx

```diff
@@ -23,13 +21,7 @@ const Login = () => {
 
   return (
     <div className="Login">
-      <h1>Log In</h1>
-      <p>Name: {userData?.username ?? 'None'}</p>
-      <p>Email: {userData?.email ?? 'None'}</p>
-      <GoogleSignIn></GoogleSignIn>
-      <SignOutButton></SignOutButton>
-      <SignIn></SignIn>
-      <SignUp></SignUp>
+      {userData ? <ProfileComponent></ProfileComponent> : <SignInComponent></SignInComponent>}
```

Comment:
i personally think using if else statement is more readable. you don't need to keep `<div className="Login">`
```
if (userData) { return <ProfileComponent/>}
else { return <SignInComponent/> }
```